doc/specs/vulkan/chapters/VK_NV_clip_space_w_scaling/vertexpostproc.txt

```diff
@@ -91,12 +91,9 @@ pname:viewportCount)#.
   * [[VUID-vkCmdSetViewportWScalingNV-None-01322]]
     The currently bound graphics pipeline must: have been created with the
     ename:VK_DYNAMIC_STATE_VIEWPORT_W_SCALING_NV dynamic state enabled
-  * [[VUID-vkCmdSetViewportWScalingNV-firstViewport-01323]]
-    pname:firstViewport must: be less than
-    sname:VkPhysicalDeviceLimits::pname:maxViewports
   * [[VUID-vkCmdSetViewportWScalingNV-firstViewport-01324]]
-    The sum of pname:firstViewport and pname:viewportCount must: be between
-    `1` and sname:VkPhysicalDeviceLimits::pname:maxViewports, inclusive
+    The sum of pname:firstViewport and pname:viewportCount must: be less
+    than or equal to slink:VkPhysicalDeviceLimits::pname:maxViewports
```

Comment:
I don't see why this part is changing. It's currently consistent with the VU in vkCmdSetViewport, and I like to keep it consistent.

Reply:
Simply because it is redundant.
It would not be obvious if the layer should report this VUID, the implicit VUID, or both for the case of `viewportCount == 0`.

Reply:
@jeffbolznv where do you stand on this - can we take it this week?

Reply:
IMO it is better to stay consistent with vkCmdSetViewports (and vkCmdBindVertexBuffers, and who knows how many other commands that follow this pattern).

Reply:
@jeffbolznv Depends where you look. It is consistent with ~6 cases and inconsistent with >20 cases (where those are two orthogonal VU statements).

It can be changed later though...

Reply:
@jeffbolznv OK, removed the VU changes (for now :evil laugh:).